storage/libradb/src/lib.rs

```diff
@@ -694,6 +694,16 @@ impl LibraDB {
         ))
     }
 
+    // ================================== Backup APIs ===================================
+    /// Gets an iterator which can yield all accounts in the state tree.
+    pub fn get_account_iter<'a> (
+        &'a self,
+        version: Version,
+    ) -> Result<Box<dyn Iterator<Item = Result<(HashValue, AccountStateBlob)>> + 'a>> {
```

Comment:
This needs to have a 'static lifetime for anything to work unfortunately.

Reply:
To be passed on to `.spawn()` for sure, but this should be usable directly with the same lifetime of the LibraDB reference, right?

Reply:
Yes for sure, I was just pointing out the issue, as to why your iterator stream didn't have a static lifetime (course you probably already knew that)

Reply:
Fixing this required a lot of movement but actually did the trick.